src/app/forms/input/input.html

```diff
@@ -0,0 +1,356 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+
+<h1>Layout - {{layout}} - {{grid ? 'grid' : 'no grid'}}</h1>
+
+<form class="clr-form clr-form-{{layout}}">
+    <div class="clr-form-control" [ngClass]="{row: grid}">
+        <label for="{{layout}}-basic" class="clr-control-label" [ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Basic input
+        </label>
+        <div class="clr-control-container" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
+            <div class="clr-input-wrapper">
+                <input type="text" id="{{layout}}-basic" placeholder="Enter value here" class="clr-input">
+                <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            </div>
+            <span class="clr-subtext">Helper Text</span>
```

Comment:
I am trying to remember what we decided about the `.clr-subtext`. Was it going to be on the `input` level or the `.clr-control-container` level? Based on that it should be in or outside of the `.clr-input-wrapper` right?

From the current implementation, it looks like if there are multiple inputs in the same row, each input can't have a subtext for it. Please correct me if I am not understanding this correctly.

Reply:
The `.clr-subtext` is always a child of `.clr-control-container`. You can have multiple `.clr-control-containers` in a single `.clr-form-control` to get multiple inputs into the same row. Each optionally has its own subtext that stays with it.

Reply:
good to know. Thanks! Do you think we can add an example which demos this?